components/Slider/src/MDCSlider.h

```diff
@@ -49,21 +49,140 @@ IB_DESIGNABLE
 
  Default color is blue.
  */
-@property(nonatomic, strong, null_resettable) UIColor *color UI_APPEARANCE_SELECTOR;
+@property(nonatomic, strong, null_resettable) UIColor *color NS_UNAVAILABLE;
+
+/**
+ Sets the color of the thumb to use for the specified state.
+
+ In general, if a property is not specified for a state, the default is to use the
+ @c UIControlStateNormal value. If the @c UIControlStateNormal value is not set, then the property
+ defaults to a system value. Therefore, at a minimum, you should set the value for the normal state.
```

Comment:
System value? Would this be better described as a material default?

Reply:
I took it from UIButton's docs. I'm happy to change to "...defaults to a material default value."

Reply:
Ah, we can't actually use Material colors as defaults, since that would change the default behavior. :-\